src/Microsoft.Data/DataFrame.BinaryOperations.cs

```diff
@@ -506,6 +504,123 @@ public DataFrame ElementwiseLessThan<T>(T value)
             }
             return retDataFrame;
         }
-        #endregion
+
+        public DataFrame RAdd<T>(T value, bool inPlace = false)
```

Comment:
`RAdd` follows our naming guidelines, but `ROr` sounds a little weird. I don't think we should name it `ReverseAdd` though